README.md

```diff
@@ -1 +1,45 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## ABOUT ME
+
+![Kagwiria Nkonge.](https://github.com/Nkonge/hw01/blob/master/Images/Profile_pic.jpg)
+
+<center>
+
+My name is Nkonge Kagwiria Regina, a graduate student (class of 2020) at Harris School of Public Policy, *studying MA International Development and Policy. I come from Kenya and I am an Obama Foundation Scholar. My policy interests are international development, security, conflict and humanitarian intervention as well as technology and education*. 
+Prior to coming to Harris my work focused on technology for development particularly the application of technology in higher education, refugee camps and hard to reach areas. I hold an MSc. in Information Systems Security and Digital Forensics.
+**I am excited to be in this class!**
+
```

Comment:
If you want these to be separate paragraphs, you need a blank line between each paragraph

```
My name is Nkonge Kagwiria Regina, a graduate student (class of 2020) at Harris School of Public Policy, *studying MA International Development and Policy. I come from Kenya and I am an Obama Foundation Scholar. My policy interests are international development, security, conflict and humanitarian intervention as well as technology and education*. 

Prior to coming to Harris my work focused on technology for development particularly the application of technology in higher education, refugee camps and hard to reach areas. I hold an MSc. in Information Systems Security and Digital Forensics.

**I am excited to be in this class!**
```

My name is Nkonge Kagwiria Regina, a graduate student (class of 2020) at Harris School of Public Policy, *studying MA International Development and Policy. I come from Kenya and I am an Obama Foundation Scholar. My policy interests are international development, security, conflict and humanitarian intervention as well as technology and education*. 

Prior to coming to Harris my work focused on technology for development particularly the application of technology in higher education, refugee camps and hard to reach areas. I hold an MSc. in Information Systems Security and Digital Forensics.

**I am excited to be in this class!**